--- upstream/gpui-pre-windows-0.3.7/src/direct_write.rs
+++ patched/gpui-pre-windows-0.3.7/src/direct_write.rs
@@ -194,7 +194,7 @@ impl DirectWriteTextSystem {
             let mut result = None;
             components
                 .factory
-                .GetSystemFontCollection(false, &mut result, true)?;
+                .GetSystemFontCollection(false, &mut result, false)?;
             result.context("Failed to get system font collection")?
         };
         let custom_font_set = unsafe { components.builder.CreateFontSet()? };
